src/main/java/com/slimebot/commands/QuoteCommand.java

```diff
@@ -30,6 +30,18 @@ public static void quote(IReplyCallback event,
 	) {
 		event.deferReply(true).queue();
 
+		if (author == event.getMember()) {
```

Comment:
I think == comparison works for JDA entities because it returns the same instance every time because of caching but it would be safer to use a .equals comparision here

Reply:
Done